omnibus/config/software/supermarket.rb

```diff
@@ -30,7 +30,7 @@
 dependency "ruby"
 dependency "runit"
 dependency "logrotate"
-dependency "fieri"
+dependency "file"
```

Comment:
With this change fieri lives as a gem whose files are located within the supermarket source tree. Fieri uses `ruby-filemagic` which is a wrapper with C extensions for `libmagic` in `file`, so supermarket gets the dependency.